CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
-## neptune-client 0.13.6
+## neptune-client 0.14.0
+
+### Features
+- Interacting with project-level metadata ([#758](https://github.com/neptune-ai/neptune-client/pull/758))
```

Comment:
Remember to add info about copy operation

Reply:
done :)